be/src/column/binary_column.cpp

```diff
@@ -62,6 +63,24 @@ void BinaryColumnBase<T>::append(const Column& src, size_t offset, size_t count)
 
 template <typename T>
 void BinaryColumnBase<T>::append_selective(const Column& src, const uint32_t* indexes, uint32_t from, uint32_t size) {
+    if (src.is_constant()) {
```

Comment:
To Support `ConstColumn` would fail to cast in line 84.

Also, this is a quick path to handle `ConstColumn`.